File: 文件夹-所有文件/getFiles.py
import os
import glob
import sys

def getFiles(dir):
    return glob.glob("%s/**/*.*" % (dir), recursive=True)


currentDir = os.path.split(os.path.realpath(sys.argv[0]))[0]
print("currentDir", currentDir)
files = getFiles(currentDir)
print(files)

File: 文件夹-所有文件/getFiles.js
const path = require("path");
const fs = require("fs");

function getFiles(root, files) {
    const dirs = fs.readdirSync(root);
    dirs.forEach(dir => {
        const fullPath = path.join(root, dir);
        const sta = fs.statSync(fullPath);
        if(sta.isFile()){
            return files.push(fullPath);
        }
        getFiles(fullPath, files);
    });

    return files;
}

const files = [];
getFiles(__dirname, files);
console.log(files);

